docs/user/tutorials/01-00-create-workload.md

```diff
@@ -0,0 +1,90 @@
+# Create a workload
+
+This tutorial explains how to create a sample HttpBin Service deployment and a sample Function.
```

Comment:
There is a lot of mention of `Function` in this docs. Unfortunately, after modularisation we will no longer have `Serverless` always on the cluster, meaning that those tutorials need to be adjusted to use a separate deployment.